library/src/main/java/org/apache/apex/malhar/lib/state/managed/Bucket.java

```diff
@@ -361,6 +405,16 @@ public Slice get(Slice key, long timeBucket, ReadSource readSource)
      */
     private BucketedValue getValueFromTimeBucketReader(Slice key, long timeBucket)
     {
+      if (bloomFilter != null) {
```

Comment:
Why the value should be inserted into the bloom filter after fetched? Do you try to avoid update with same value? Could you explain more here?

What I usage is different here. What I try to avoid to is unnecessary get as there are much more getting than setting. 

The reason why decrease accurate is due to there are much more put() in all than put() to the file. and each put add flags to the bloom filter,